### DISCUSSION TITLE
Office LTSC專業增強版 2021 (大量授權)無法啟用

### DISCUSSION THREAD

**Discussion by Sy-342** [YerongAI/Office-Tool#918] [Q&A] [answered]
一直跑0x8007007B錯誤，看了解法跟0x8007000D一樣，
時間OK、網路OK、KMS主機實在不會確認...
重設軟體保護也做了....
但還是一直無法啟用激活，所以快過期QQ

**Comment by YerongAI** [comment, marked as answer]
重設軟體保護之後，換用 kms.loli.best，并在主機后填入連接埠：kms.loli.best:1688。完成之後再嘗試啟用授權。

**Reply by Sy-342** [reply]
謝謝，設定好主機後就啟用成功了！
